action.yml

```diff
@@ -27,32 +21,3 @@ runs:
         file_path="$GITHUB_ACTION_PATH/settings.xml"
         
         echo "path=${file_path}" >> $GITHUB_OUTPUT
-
```

Comment:
On [line 14](https://github.com/pentaho/actions-common/pull/31/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R14), that `if` condition, is no longer needed, right?

Reply:
Removed